source/module_cell/read_pp_upf201.cpp

```diff
@@ -230,19 +254,37 @@ int Pseudopot_upf::read_pseudo_upf201(std::ifstream &ifs)
 		ifs >> word; //number of beta
 	}
 
-	ModuleBase::GlobalFunc::SCAN_BEGIN(ifs, "<PP_DIJ");
-	ModuleBase::GlobalFunc::READ_VALUE(ifs, word);  // type size columns
+	if (ModuleBase::GlobalFunc::SCAN_BEGIN(ifs, "<PP_DIJ"))
+	{
+		ModuleBase::GlobalFunc::READ_VALUE(ifs, word);  // type size columns
 
-	this->nd = nbeta * nbeta;
-	for(int i=0;i<nbeta;i++)
+		this->nd = nbeta * nbeta;
+		for(int i=0;i<nbeta;i++)
+		{
+			for(int j=0;j<nbeta;j++)
+			{
+				ifs >> dion(i,j);
+				if ( i != j  && dion(i,j) != 0.0 )
+				{
+					std::cout << " error: for i != j, Dij of Pseudopotential must be 0.0 " << std::endl;
+					exit(1);
+				}
+			}
+		}
+	}
+	else if ( ModuleBase::GlobalFunc::SCAN_BEGIN(ifs, "<PP_DIJ>"))
 	{
-		for(int j=0;j<nbeta;j++)
+		this->nd = nbeta * nbeta;
+		for(int i=0;i<nbeta;i++)
 		{
-			ifs >> dion(i,j);
-			if ( i != j  && dion(i,j) != 0.0 )
+			for(int j=0;j<nbeta;j++)
 			{
-				std::cout << " error: for i != j, Dij of Pseudopotential must be 0.0 " << std::endl;
-				exit(1);
+				ifs >> dion(i,j);
+				if ( i != j  && dion(i,j) != 0.0 )
+				{
+					std::cout << " error: for i != j, Dij of Pseudopotential must be 0.0 " << std::endl;
```

Comment:
Maybe use `WARNING_QUIT`here is better? @dyzheng

Reply:
Yes, please use the global function rather than raw cout and exit code.

Reply:
OK, I have replace `exit` with `WARNING_OUIT`.